src/pg/test/sql/select.test.js

```diff
@@ -0,0 +1,86 @@
+import { selectByArgs, selectByIds } from '../../sql/select.js';
```

Comment:
The tests are great! Thank you!

One minor point on the naming, I prefer to use names that are easily grep-able, so I can specify them on the command line.

Here I might suggest:
- describe('pg_select')
- tests:
  - byArgs_first
  - byIds
  - byArgs_order_first
  - byArgs_order_before